### ISSUE TITLE
Accessibility checks

### ISSUE THREAD

**Issue by rosselton** [NewcastleRSE/HNCancer#29]
The site should be checked against accessibility tools Lighthouse and Wave to ensure W3C compliance.

**Comment by gmschroe** [comment]
I think the download button is currently a div - recommend replacing it anyway with a download button in a toolbar (#83)
